main.tf

```diff
@@ -18,3 +18,11 @@ resource "aws_s3_bucket" "accuricsbucketdemo" {
     Name = "bucketdemo"
   }
 }
+
+resource "aws_vpc" "<resource_name>" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure virtual private cloud (VPC) is configured for AWS EC2 instances**
 
**Vulnerability**
By default AWS EC2 instances are launched in default VPC which is not recommended for production grade. AWS VPCs provides the controls to facilitate a formal process for approving and testing all network connections and changes to the firewall and router configurations.
 
**Remediation**
Migrate from EC2 classic to VPCs. 1: Create a VPC. 2: Configure Your Security Group. 3: Create an AMI from Your EC2-Classic Instance. 4: Launch an Instance Into Your VPC.

source: accurics-stageone